b_server.md

```diff
@@ -15,245 +19,372 @@ If the server is responding returns:
 ```
 {% include starsep.html %}
 
-### Onboarding a new user
-This is an API to create a signup link that BobCo can send to Alice, inviting her to take charge of the contact information he has about her, and set her preferences about what he can do with it.
+<a name="create-organization"></a>
+### Creating a new organization
+
+Endpoint:
+```
+POST http://b-api.jlinc.test/api/signup
+```
+
+Generates a signup token and sends an email to the given email address with a verification link.
 
-_POST https://sandbox.b.jlinclabs.net/api/userdata/new_user_
+Requires either a `stripeToken` id (see [Stripe API docs: tokens](https://stripe.com/docs/api/tokens)) or an `inviteToken`. An `inviteToken` is created by running the `createOrganizationAdminInviteToken` script on the B Server. It should then added as a param to an invite url that can be sent to a prospective organization admin, example: `http://b-portal.jlinc.test?it=V7i1ldl8HoeVFBDtL9paUbS2kFl3rlB`. If an `inviteToken` is provided, it should first be [verified](#verify-invite-token) before posting to `api/signup`.
 
+_Post body:_
 ```json
 {
- "apikey": "bobco",
- "apisecret": "670b5bba405d62961751a6b97dbfaf5c",
- "data": {
-	"email": "alice@example.com",
-	"firstname": "Alice",
-	"lastname": "McPerson",
-	"fullname": "Alice McPerson",
-	"mailingstreet": "123 Main Street",
-	"mailingcity": "Oakland",
-	"mailingstate": "CA",
-	"mailingpostalcode": "01234",
-	"mailingcountry": "US",
-	"homephone": "555-111-4444",
-	"mobilephone": "555-111-2222"
- }
+  "email": "neworg@example.com",
+  "stripeToken": "tok_1CR5JRFNXsZIJI8lCl1l04S4",
+  "inviteToken:": "d1830fcda87ae8993039694e6f7aea95"
 }
 ```
-The `apikey` and `apisecret` are required for authentication.
-
-Under the `data` key, the `email` is required, and must be unique in BobCo's account. All other data fields are optional. Any other fields that might be included will be ignored. From time to time new optional data fields may be added.
 
-On success a signup link will be returned:
+A successful response returns success `true`.
 
+_Response:_
 ```json
 {
- "success": true,
- "signup": "https://alpha.userprefs.com/regauth/65c29f1ff4975dff"
+ "success": true
 }
 ```
 
-This link should be sent to Alice to register her account with BobCo.
-
-{% include starsep.html %}
+Example signup link sent in the verification email: `http://b-portal.jlinc.test/signup/eV7i1ldl8HoeVFBDtL9paUbS2kFl3rlB`.
```

Comment:
not sure what url to use here since we do not have a sandbox b portal. @victorgrey what should the sandbox b portal domain be?

Reply:
I went with sandbox.portal.jlinclabs.net